.moban.yaml

```diff
@@ -0,0 +1,32 @@
+overrides: coala.yaml
+
+name: documentation
+contact: coala.analyzer@gmail.com
```

Comment:
a lot of these are not necessary yet, so should be in this PR.

Reply:
should = should not

Reply:
This probably also needs an update as new updates to the templates have been merged